metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_query.py

```diff
@@ -747,7 +760,7 @@ def table_upstreams_with_column_lineage(
                     'upstream_object_domain', h.upstream_table_domain
                 )
             ) AS "UPSTREAM_TABLES",
-            ARRAY_AGG(
+            ARRAY_UNIQUE_AGG(
```

Comment:
This was missed when changes were made for this comment - https://github.com/datahub-project/datahub/pull/7710#discussion_r1154836799

Earlier it was one row per table. With join on column_lineage_history, this needed to be updated.